### PR TITLE
Misc Cleanup 2

### PR DESCRIPTION
**Pull request by engineer124**
Even more brackets, this time with a few `if (1) {}`'s removed

### CODE REVIEW

File: src/overlays/actors/ovl_En_Fg/z_en_fg.c [hunk 1]
```diff
@@ -461,26 +471,27 @@ void EnFg_DrawDust(PlayState* play, EnFgEffectDust* dustEffect) {
...
+        gDPPipeSync(POLY_XLU_DISP++);
+        Matrix_Translate(dustEffect->pos.x, dustEffect->pos.y, dustEffect->pos.z, MTXMODE_NEW);
+        Matrix_ReplaceRotation(&play->billboardMtxF);
+        Matrix_Scale(dustEffect->xyScale, dustEffect->xyScale, 1.0f, MTXMODE_APPLY);
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        index = 0.5f * dustEffect->timer;
+        gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[index]));
+        gSPDisplayList(POLY_XLU_DISP++, object_fr_DL_00B338);
     }
     CLOSE_DISPS(play->state.gfxCtx);
```
Comment (tom-overton) [inline src/overlays/actors/ovl_En_Fg/z_en_fg.c:496]: Maybe add a blank line here? Normally we have a blank line before `CLOSE_DISPS`, but with the brace here, I don't know what you think.

**Review by tom-overton — APPROVED** [review]
Nice work! Just one thing I noticed.

File: src/code/z_collision_btltbls.c [hunk 2]
```diff
@@ -790,7 +790,7 @@ DamageTable sDamageTablePresets[] = {
  * is out of range.
  */
 DamageTable* DamageTable_Get(s32 index) {
-    if (index < 0 || index > ARRAY_COUNT(sDamageTablePresets) - 1) {
+    if ((index < 0) || (index > (ARRAY_COUNT(sDamageTablePresets) - 1))) {
```
Comment (AngheloAlf) [inline src/code/z_collision_btltbls.c:793]: Now that you are here
```suggestion
    if ((index < 0) || (index >= ARRAY_COUNT(sDamageTablePresets))) {
```

File: src/code/z_eff_blure.c [hunk 3]
```diff
@@ -753,53 +753,51 @@ void EffectBlure_DrawSimpleVertices(GraphicsContext* gfxCtx, EffectBlure* this,
...
+            if (!(this->flags & 4)) {
+                continue;
+            }
+
+            sp1B0.x = ((f32)vtx[4 * i + 0].v.ob[0] + (f32)vtx[4 * i + 1].v.ob[0]) * 0.5f;
+            sp1B0.y = ((f32)vtx[4 * i + 0].v.ob[1] + (f32)vtx[4 * i + 1].v.ob[1]) * 0.5f;
+            sp1B0.z = ((f32)vtx[4 * i + 0].v.ob[2] + (f32)vtx[4 * i + 1].v.ob[2]) * 0.5f;
+            sp1A4.x = ((f32)vtx[4 * i + 2].v.ob[0] + (f32)vtx[4 * i + 3].v.ob[0]) * 0.5f;
+            sp1A4.y = ((f32)vtx[4 * i + 2].v.ob[1] + (f32)vtx[4 * i + 3].v.ob[1]) * 0.5f;
+            sp1A4.z = ((f32)vtx[4 * i + 2].v.ob[2] + (f32)vtx[4 * i + 3].v.ob[2]) * 0.5f;
```
Comment (AngheloAlf) [inline src/code/z_eff_blure.c:776]: Does doing ` / 2.0f` matches here? This construct makes me think on calculating an average

File: src/code/z_scene.c [hunk 4]
```diff
@@ -89,7 +89,7 @@ s32 Object_GetIndex(ObjectContext* objectCtx, s16 objectId) {
     s32 i;
 
     for (i = 0; i < objectCtx->num; i++) {
-        if ((objectCtx->status[i].id < 0 ? -objectCtx->status[i].id : objectCtx->status[i].id) == objectId) {
+        if (((objectCtx->status[i].id < 0) ? -objectCtx->status[i].id : objectCtx->status[i].id) == objectId) {
```
Comment (AngheloAlf) [inline src/code/z_scene.c:92]: Isn't this an `ABS_ALT`?

File: src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c [hunk 5]
```diff
@@ -65,7 +65,7 @@ s32 ObjKibako2_ContainsSkulltula(ObjKibako2* this, PlayState* play) {
     if ((u16)actorSpawnParam & 3) {
         flag = ((actorSpawnParam & 0x3FC) >> 2) & 0xFF;
     }
-    return !(flag >= 0 && Flags_GetTreasure(play, flag));
+    return !((flag >= 0) && Flags_GetTreasure(play, flag));
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c:68]: I assume this doesn't match, right?
```suggestion
    return (flag < 0) || !Flags_GetTreasure(play, flag);
```

File: src/overlays/actors/ovl_En_Fg/z_en_fg.c (hunk 1 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Fg/z_en_fg.c]: Yeah, I've been putting a line before even after a brace. Just missed this one

File: src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c (hunk 5 above)
Comment (engineer124) [reply src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c:68]: Unfortunately not :(

File: src/code/z_snap.c [hunk 6]
```diff
@@ -71,15 +68,20 @@ s32 Snap_RecordPictographedActors(PlayState* play) {
...
             // If actor is recordable, run its validity function and record if valid
-            if (seen) {
-                pictoActor = (PictoActor*)actor;
-                if (pictoActor->validationFunc != NULL) {
-                    if ((pictoActor->validationFunc)(play, actor) == 0) {
-                        validCount++;
-                    }
+            pictoActor = (PictoActor*)actor;
+            if (pictoActor->validationFunc != NULL) {
+                if ((pictoActor->validationFunc)(play, actor) == 0) {
```
Comment (hensldm) [inline src/code/z_snap.c:83]:
```suggestion
                if (pictoActor->validationFunc(play, actor) == 0) {
```

File: src/overlays/actors/ovl_En_Daiku/z_en_daiku.c [hunk 7]
```diff
@@ -197,17 +197,18 @@ void func_809438F8(EnDaiku* this, PlayState* play) {
     }
 
     if (this->unk_278 == ENDAIKU_PARAMS_FF_3) {
-        f32 sq, abs;
+        f32 xzDist;
+        f32 abs;
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Daiku/z_en_daiku.c:201]:
```suggestion
        f32 yaw;
```
Note I actually fix this in #1205 (https://github.com/zeldaret/mm/pull/1205#discussion_r1133139110), but it is fine here. I like `xzDist` over `dist` anyways.

**Review by hensldm — COMMENTED** [review]
A couple nits

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/actors/ovl_En_Daiku/z_en_daiku.c (hunk 7 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Daiku/z_en_daiku.c]: I knew I had seen something similar before, wasn't sure if it was already merged and this was just a similar instance or not

**Review by hensldm — APPROVED** [review]

**Pull request merged**